### PR TITLE
Add `.dockerignore` file to `cog init` template

### PR DESCRIPTION
**Pull request by mattt**
Resolves https://github.com/replicate/cog/issues/209

### CODE REVIEW

File: pkg/cli/init-templates/.dockerignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# https://docs.docker.com/engine/reference/builder/#dockerignore-file
+.DS_Store
+.git
+.github
+.gitignore
```
Comment (zeke) [inline pkg/cli/init-templates/.dockerignore:5]: Are there some other good defaults we could add here?

cc @replicate/models

**Review by zeke — APPROVED** [review]
Looks good to me. 👍🏼 

It would be nice to get some other good defaults in the ignore file, if there are some no-brainer file types that fit the bill.

File: pkg/cli/init-templates/.dockerignore (hunk 1 above)
Comment (mattt) [reply pkg/cli/init-templates/.dockerignore:8]: @zeke I just updated this based on [other `.dockerignore` files in our org](https://github.com/search?q=org%3Areplicate+path%3A.dockerignore&type=code).

**Comment by zeke** [comment]
Nice!

**Pull request merged**